test/scripts/phyx2owl.js

```diff
@@ -40,14 +40,14 @@ describe(PHYX2OWL_JS, function () {
     const OWL_FILE = path.resolve(__dirname, '../examples/correct/brochu_2003.owl');
     const EXPECTED_OWL_FILE = path.resolve(__dirname, '../examples/correct/brochu_2003.jsonld');
 
-    // If there is already a '../examples/brochu_2003.json' file, we should delete it.
+    // If there is already a '../examples/brochu_2003.owl' file, we should delete it.
```

Comment:
I assume this is because it gets generated by the tests, right? If so, it seems to be this should be done in a teardown and/or setup script, not somewhere obscure.

Reply:
Yup! I've opened an issue to figure this out (https://github.com/phyloref/phyx.js/issues/82). I've been trying to work around an issue with the JPhyloRef JAR download -- that's something that would also be nice to move into a setup block somewhere.